ziti-controller/subcmd/delete_sessions.go

```diff
@@ -0,0 +1,137 @@
+/*
+	Copyright NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package subcmd
+
+import (
+	"github.com/michaelquigley/pfxlog"
+	"github.com/openziti/fabric/controller"
+	"github.com/openziti/ziti/common/version"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"go.etcd.io/bbolt"
+)
+
+func init() {
+	root.AddCommand(deleteSessionsCmd)
+}
+
+var deleteSessionsCmd = &cobra.Command{
+	Use:   "delete-sessions <config>",
+	Short: "Delete all API Sessions and Edge Sessions, controller must be shutdown",
+	Args:  cobra.ExactArgs(1),
+	Run:   deleteSessions,
+}
+
+const (
+	ApiSessionBucketName = "apiSessions"
+	SessionBucketName = "sessions"
+)
+
+func deleteSessions(_ *cobra.Command, args []string) {
+	logrus.WithField("version", version.GetVersion()).
+		WithField("go-version", version.GetGoVersion()).
+		WithField("os", version.GetOS()).
+		WithField("arch", version.GetArchitecture()).
+		WithField("build-date", version.GetBuildDate()).
+		WithField("revision", version.GetRevision()).
+		Info("removing API Sessions and Edge Sessions from ziti-controller")
+
+	if config, err := controller.LoadConfig(args[0]); err == nil {
+
+		apiSessionBucketExists := false
+		sessionBucketExists := false
+
+		logger := pfxlog.Logger()
+
+		defer func() {
+			_ = config.Db.Close()
+		}()
+
+		err = config.Db.View(func(tx *bbolt.Tx) error {
+			root := tx.Bucket([]byte("ziti"))
+
+			if root == nil {
+				return errors.New("root 'ziti' bucket not found")
+			}
+
+			apiSessionBucket := root.Bucket([]byte(ApiSessionBucketName))
+
+			if apiSessionBucket == nil {
+				logger.Info("API Session bucket does not exist, skipping, count is: 0")
```

Comment:
Stylistically, golang in general, and ziti usually, have typically used all lowercase in log messages.

Reply:
Moved them all to lower case